### ISSUE TITLE
Issue with porting guide

### ISSUE THREAD

**Issue by SecurityOfThings2016** [ARMmbed/uvisor#134]
upon execution of command git clone --recursive git@github.com:ARMMbed/uvisor.git throws following error:
Cloning into 'uvisor'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by SecurityOfThings2016** [comment]
I managed  to find the workaround for this problem. I did git clone of https://github.com/ARMmbed/uvisor.git into uvisor folder and git clone of https://github.com/ARMmbed/uvisor-lib.git into uvisor/release folder. with this setup, I managed to run instructions in porting guide to build my version of uvisor-lib. It appears that there is permission issues with repositories to use with git clone --recursive https://github.com/ARMmbed/uvisor.git
